test/unit/actions-test.js

```diff
@@ -79,6 +81,48 @@ describe('Denali.Action', function() {
     });
   });
 
+  describe.only('an app with a default JSON-API serializer', function() {
+    it('should not invoke the default serializer if serializer=false in an action', function() {
+      let success = false;
+      let mock = mockReqRes({
+        request: {
+          headers: {
+            'content-type': 'application/json',
+            'transfer-encoding': 'chunked',
+          },
+          method: 'post',
+          url: '/a-path',
+          body: { my: 'post-data' }
+        }
+      });
+      class IssuesAction extends Action {
+        serializer = false;
+        run() {
+          success = true;
+          expect(success).to.be.true();
```

Comment:
Not sure if this is the best approach - defining success and then asserting it's value won't actually check anything. And unlike QUnit, Mocha isn't aware of assertions, so it doesn't care. If this issue regresses and `serializer = false` still picks a serializer, this code branch will never be reached, and it seems like the test will not fail when it should.

I would suggest two things:

1. Register a mock serializer that throws if you call `parse()` (literally just `{ parse() { throw new Error('...'); } }`). That way, if this code regresses and a serializer is chosen despite `serializer = false`, it will error out.
2. Update the `Router#handle()` method to return a promise that resolves when `action.run()` does. Because `handle()` is async, our assertions will be async, and mocha needs to know when to consider the test "finished". Because we don't return any promises here, mocha will assume the test is sync, and since no errors were thrown, it will pass. If we change that so `handle()` does return a Promise, then we can add an `expect(success).to.be.true()` once it resolves, ensuring that not only did the serializer _not_ get call, but `action.run()` did.